multiqc/qualimap/__init__.py

```diff
@@ -28,10 +30,16 @@ def __init__(self):
              is a platform-independent application to facilitate the quality control of alignment \
              sequencing data and its derivatives like feature counts.</p>'
 
+        self.parsed_stats = defaultdict(dict)
+
         # Find QualiMap reports
         qualimap_raw_data = {}
         for root, dirnames, filenames in os.walk(config.analysis_dir, followlinks=True):
-            if 'genome_results.txt' in filenames and 'raw_data_qualimapReport' in dirnames:
+            raw_data_dir = 'raw_data'
+            for d in dirnames:
+                if raw_data_dir in d:
+                    raw_data_dir = d
+            if 'genome_results.txt' in filenames and raw_data_dir in dirnames:
```

Comment:
Why do you need to look for a directory at all? Why can't you just look for a file called `genome_results.txt`?